lib/blpapi-wrapper.ts

```diff
@@ -320,12 +320,10 @@ export class Session extends events.EventEmitter {
         this.openService(uri).then((): void => {
             log(util.format('Request: %s|%d', requestName, correlatorId));
             trace(request);
-            if (!(requestName in REQUEST_TO_RESPONSE_MAP)) {
-                throw new Error('Unknown request type ' + requestName);
-            }
-            var responseEventName = REQUEST_TO_RESPONSE_MAP[requestName];
             this.session.request(uri, requestName, request, correlatorId);
-            this.listen(responseEventName,
+            assert(requestName in REQUEST_TO_RESPONSE_MAP,
+                   util.format('Request, %s, not handled', requestName));
```

Comment:
`this.session.reqeust()` will already throw an exception and error appropriately.  So the `assert` is valid to ensure that our internal imp-details of the mapping is correct.